acquia_lift.module

```diff
@@ -1,7 +1,6 @@
 <?php
 
 /**
- * @file
```

Comment:
https://www.drupal.org/node/2665744
https://www.drupal.org/node/2304909

So @file is no longer required in D8.2 and somebody also said in one of above comments that the coding standards doc is getting updated.

The actual reasoning is that it is one-file-one-class in Symfony/D8 anyway, and the directory structure & file name should match namespace & class name; so, there is really no advantage to have @file like that any more. I personally also found myself making mistakes over it often, so I promptly removed.